qemu/tests/qemu_guest_agent.py

```diff
@@ -231,23 +232,29 @@ def setup(self, test, params, env):
             vm = self.env.get_vm(params["main_vm"])
             vm.verify_alive()
             self.vm = vm
-            session = self._get_session(params, self.vm)
+        session = self._get_session(params, self.vm)
```

Comment:
Why move it out of the judgement?

Reply:
The previous if has a problem with existed self.vm. Because, if sefl.vm exists, it will not get session.

Reply:
@YanhuiMa I see, you are right.